src/src/RobotMove.java

```diff
@@ -0,0 +1,22 @@
+public class RobotMove {
+	RobotConnection robotConnection;
```

Comment:
add the method here:
public static void moveRobot(RobotConnectionManager robotConnectionManager, int toX, int toY)